ios/PurchasesHybridCommon/PurchasesHybridCommon.xcodeproj/project.pbxproj

```diff
@@ -563,10 +563,12 @@
 			baseConfigurationReference = 460EB988CBB68111D4CF19A2 /* Pods-PurchasesHybridCommon.debug.xcconfig */;
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
+				CODE_SIGN_STYLE = Manual;
```

Comment:
ᕙ(⇀‸↼‶)ᕗ

Reply:
I think it has to be manual, right?

Reply:
we should be able to use the other projects as a reference